### DISCUSSION TITLE
How to add suggestions to vscode?

### DISCUSSION THREAD

**Discussion by wsw70** [quasarframework/quasar#13765] [General - Components / Directives / etc] [answered]
I usually use PyCharm and get suggestions for all elements of my code, including classes:

![image](https://user-images.githubusercontent.com/1345886/175325813-fde83f3c-fa89-4b0b-bcd7-93ec6eacbdfd.png)

I wanted to give a try to vscode and while tags suggestions work, classes are not suggested:

![image](https://user-images.githubusercontent.com/1345886/175325582-23e4de59-b53a-4730-89cc-ce2abc7c3142.png)

**Which settings should I use do to have Quasar class suggestions?**

**Comment by pratik227** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/34883558/175328847-7404df9e-7db2-4f8f-af4f-de1c4acb9137.png)

**Reply by wsw70** [reply]
Thank you! I had to add `vue` to the processed files:

```json
{
  "html-css-class-completion.remoteStyleSheets": ["https://cdn.jsdelivr.net/npm/quasar@latest/dist/quasar.prod.css"],
  "html-css-class-completion.includeGlobPattern": "**/*.{css,scss,sass,html,vue}"
}
```
